stack/database.py

```diff
@@ -117,8 +144,8 @@
     DBInstanceClass=Ref(db_class),
     DBInstanceIdentifier=Ref(AWS_STACK_NAME),
     Engine="postgres",
-    EngineVersion="9.4.5",
-    MultiAZ=True,
+    EngineVersion=Ref(db_engine_version),
+    MultiAZ=Ref(db_multi_az),
```

Comment:
I undrestand the cost issue, but then the parameter should also create a single `AZ` cluster.

The multi `AZ` `RDS` does not trigger a `master / slave` replica setup, it synced `multi-masters`. c.f.: http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.MultiAZ.html

> DB instances using Multi-AZ deployments may have increased write and commit latency compared to a Single-AZ deployment, due to the synchronous data replication that occurs.  

So the intenal `DNS` resolution will always drive you to the same `AZ` if available. 

No?